backend/app/api/routes/content.py

```
@@ -833,3 +1759,306 @@ def deactivate_share_link_endpoint(
+    from app.models.content import ProcessingJob
+    from sqlmodel import select  # 添加缺失的导入
+
+    # 首先从AIConversation表获取最新的AI分析结果
+    conversations = session.exec(
+        select(AIConversation)
+        .where(AIConversation.content_item_id == content_id)
+        .order_by(AIConversation.created_at.desc())
+    ).all()
+
+    ai_analysis = {}
+    
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)